modules/nf-core/picard/collecthsmetrics/main.nf

```diff
@@ -8,9 +8,9 @@ process PICARD_COLLECTHSMETRICS {
         'quay.io/biocontainers/picard:2.27.4--hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(bam)
-    path fasta
-    path fai
+    tuple val(meta), path(bam), path(bai)
+    tuple val(meta2), path(fasta)
+    tuple val(meta2), path(fai)
```

Comment:
```suggestion
    tuple val(meta3), path(fai)
```

Reply:
shouldn't the fasta and it's index have the same meta? It's like that in other subwfs I think

Reply:
they're different files.
I'd say the fasta can have a meta describing species + version + file, which then would be different from the fai, that could on top have info on the version of the tools that build it

Reply:
Plus, I'm concerned about collapsing meta maps, even if it should not matter much in that case

Reply:
Fixed it!